wren-ai-service/src/pipelines/generation/sql_regeneration.py

```
@@ -22,6 +22,8 @@
 )
 from src.pipelines.retrieval.sql_functions import SqlFunction
 from src.utils import trace_cost
+from src.web.v1.services import Configuration
+from src.web.v1.services import Configuration
```

Ruff (0.12.2)

Redefinition of unused `Configuration` from line 25

Remove definition: `Configuration`

(F811)